FrontPage/AppDelegate.swift

```diff
@@ -7,6 +7,8 @@ let syncUrl = config.getConfiguration("sync-app")
 let syncConfig = syncUrl.config ?? ["error": JSONValue.bool(false)]
 let syncWs = syncConfig["websocketUrl"]?.getString() ?? "error";
 
+var pushConfig = UnifiedPushConfig()
```

Comment:
similar to my comments in #6, we should avoid using global variables wherever we can.